reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md

```diff
@@ -145,9 +155,12 @@ Accept wildcard characters: False
 
 Specifies the hashing algorithm that Windows uses to compute the digital signature for the file.
 
-For Windows PowerShell 3.0, the default is SHA256, which is the Windows default hashing algorithm.
-For Windows PowerShell 2.0, the default is SHA1.
-Files that are signed with a different hashing algorithm might not be recognized on other systems.
+For PowerShell 3.0, the default is SHA256, which is the Windows default hashing algorithm. For
+PowerShell 2.0, the default is SHA1. Files that are signed with a different hashing algorithm might
+not be recognized on other systems. Which algorithms are supported depends on the version of the
+operating system.
+
+For a list of possible values, see [HashAlgorithmName Struct ](/dotnet/api/system.security.cryptography.hashalgorithmname?view=netframework-4.7.2#properties).
```

Comment:
Given the framework support at that link, are all the options supported in PS3?

Reply:
Yeah, its the same list.